source/blog/2016-07-22-brighton-ruby-conference-2016.html.markdown

```diff
@@ -14,7 +14,7 @@ authors:
 [Brighton Ruby](http://brightonruby.com/) is unsurprisingly a conference about Ruby based in Brighton, and organised by [@AndyCroll](https://twitter.com/andycroll). Being quite new to both Ruby and to Rails, a Ruby conference seemed like the perfect place to pickup a few tricks of the trade and see what's next for the language and community.
 
 ![The Ruby Conf Crew](/assets/images/blog/Ruby_Team.png)
-*The Ubxd Ruby Conf team, from left to right: [Neil](https://unboxed.co/people#neil-van-beinum), Jon, [Murray](https://unboxed.co/people#murray-steele), [Chris](https://unboxed.co/people#chris-holmes), [Crystal](https://unboxed.co/people#chris-carter), Cale, [Ben](https://unboxed.co/people#ben-wong), and [Tom](https://unboxed.co/people#tom-sabin)*
+*The Ubxd Ruby Conf team, from left to right: [Neil](https://unboxed.co/people#neil-van-beinum), Jon, [Murray](https://unboxed.co/people#murray-steele), Chris, Cale, [Ben](https://unboxed.co/people#ben-wong), and [Tom](https://unboxed.co/people#tom-sabin)*
```

Comment:
Was this done with the script?  Perhaps it's a case of the matcher being too greedy. Looks like it matched [Chris](...#chris-carter) where that ... happened to include lots of other content.